src/pybind/mgr/volumes/fs/operations/volume.py

```diff
@@ -268,40 +92,28 @@ def list_volumes(mgr):
         result.append({'name': f['mdsmap']['fs_name']})
     return result
 
-@contextmanager
+
```

Comment:
this still needs to be a context manager since it's supposed to be used via "with" statement.

Reply:
sorry, I didn't commit that the first time around.